torch/csrc/jit/passes/quantization.cpp

```diff
@@ -735,13 +737,13 @@ void checkGetQParamsResult(const IValue& qparams) {
   }
 }
 
-QParamMap InsertQuantDeQuantHelper::getQParamMap(
+std::tuple<QScheme, QParamMap> InsertQuantDeQuantHelper::getQSchemeAndQParamMap(
```

Comment:
It doesn't look like we're ever using the first element of the tuple. Do we really need it?

Reply:
It's used in next PR :)